frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/disk-wrapper.ts

```diff
@@ -66,17 +46,11 @@ export class DiskWrapper extends ObjectWithTypePropertyWrapper<V1Disk, DiskType>
   isFirstBootableDevice = () => this.getBootOrder() === 1;
 
   hasBootOrder = () => this.getBootOrder() != null;
-}
 
-export class MutableDiskWrapper extends DiskWrapper {
-  public constructor(disk?: V1Disk, copy = false) {
-    super(disk, { copy });
+  protected sanitize(type: DiskType, { bus }: CombinedTypeData) {
```

Comment:
each wrapper should implement its own sanitize function

Reply:
> each wrapper should implement its own sanitize function

In case each wrapper's `sanitize` will be different in some (wrapper-specific) way, maybe we should make it an `abstract` method - assuming we'd never want to use the fallback "clone deep" implementation for any of the wrappers.

Reply:
I would keep for now
- we can default to it in the child class
- could be used by a type which would have the same attributes for each instance

Reply:
OK, let's keep it for now.